modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view_content_dashboard_item_types.jsp

```diff
@@ -72,14 +71,11 @@ ContentDashboardItemTypeItemSelectorViewDisplayContext contentDashboardItemTypeI
 		var arr = [];
 
 		allSelectedElements.each(function () {
-			var row = this.ancestor('tr');
```

Comment:
As you said, if we change the page we lose the `tr` information and we only have the hidden inputs with the selected data in `value` property. Where does this input fill out with the selected data?

Reply:
Hey @dgarciasarai !!!

I have provided this info in the commit message, please take a look, and if you need more data, let me know, so I can add more info.

Thanks!!!

Reply:
Yes, I saw it, and I was debugging on `seach_container_select.js` to see what has elements array (fill with the data from the `value` property of the hidden input). I just don't know where this hidden input filled with the value data before sending it to the `search_container_select.js`. I'm trying to understand the whole cycle and learn for the future :)

Reply:
Yeahh!!, exactly, that is the reason we need to delete the data, and use the primaryKey (value) instead, because the data is not filled.